digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionCalculator.scala

```diff
@@ -67,7 +67,7 @@ object ProjectSectionCalculator {
     */
   def assignMValues(projectLinks: Seq[ProjectLink]): Seq[ProjectLink] = {
     val (terminated, others) = projectLinks.partition(_.status == LinkStatus.Terminated)
-    val (newLinks, nonTerminatedLinks) = others.partition(_.status == LinkStatus.New)
+    val (newLinks, nonTerminatedLinks) = others.partition(l => l.status == LinkStatus.New || l.status == LinkStatus.Transfer)
```

Comment:
Transfer shouldn't be given completely new addresses. (may produce incorrect results if address length doesn't match to geometry length)
nonTerminatedLinks should be renamed as addressedLinks for clarity.